infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -319,13 +482,18 @@ public long getLogHead() {
         return runtime.getAddressSpaceView().getTrimMark().getSequence();
     }
 
+    /**
+     * All types of metadata stored in the corfu table, those corresponding strings are used as keys to access
+     * the real values.
+     */
```

Comment:
If possible, could we have some comments to elaborate on those types?
I feel confused if they are based on corfu log sequence number, and what does SEQ_NUM mean?...

Reply:
It is at the beginning of  the file.